node/src/Transaction.ts

```diff
@@ -1681,6 +1682,22 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
     public lolwut(options?: LolwutOptions): T {
         return this.addAndReturn(createLolwut(options));
     }
+
+    /**
+     * Loads a library to Valkey.
+     *
+     * See https://valkey.io/commands/function-load/ for details.
+     *
+     * since Valkey version 7.0.0.
+     *
+     * @param libraryCode - The source code that implements the library.
+     * @param replace - Whether the given library should overwrite a library with the same name if it
+     *     already exists.
+     * Command Response - The library name that was loaded.
```

Comment:
nit: some commands put an empty line between the last `@param` and the `Command Response` sections, and some do not. Do we want to align on this as well?